http/src/main/java/io/micronaut/http/filter/FilterOrderProvider.java

```diff
@@ -23,7 +23,9 @@
  *
  * @author James Kleeh
  * @since 1.0
+ * @deprecated Deprecated without replacement. This class is unused and will be removed in 4.0.0.
```

Comment:
Can you clarify in the javadoc what users should use instead?

Reply:
there is no replacement, i dont know what this class was for and how it was supposed to be used. it was never referenced anywhere